common/src/main/java/hunternif/mc/impl/atlas/client/gui/GuiAtlas.java

```diff
@@ -1012,14 +1010,14 @@ public void render(MatrixStack matrices, int mouseX, int mouseY, float par3) {
             Identifier tile = biomeData.getTile(pos.x, pos.z);
 
             if (tile == null) {
-                drawTooltip(Arrays.asList(new LiteralText(coords), new LiteralText(chunks)), textRenderer);
+                drawTooltip(Arrays.asList(Text.translatable(coords), Text.translatable(chunks)), textRenderer);
             } else {
                 String texture_set = TileTextureMap.instance().getTextureSet(tile).name.toString();
                 drawTooltip(Arrays.asList(
-                                new LiteralText(coords),
-                                new LiteralText(chunks),
-                                new LiteralText("Tile: " + tile),
-                                new LiteralText("TSet: " + texture_set)),
+                                Text.translatable(coords),
+                                Text.translatable(chunks),
+                                Text.translatable("Tile: " + tile),
+                                Text.translatable("TSet: " + texture_set)),
```

Comment:
LiteralText isn´t something that should be translated. Isn´t there a `Text.literal` in place for that?

Reply:
Oops! must have been a find-replace issue. Went through the diff and fixed all of these in a61ff06